examples/mobile/UIComponents/components/view-controls/main-tab/main-tab-5-items.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page">
+		<link href="./main-tab-sample.css" rel="stylesheet" />
+		<div class="ui-content ui-tabs" data-change-duration="0">
+			<div class="ui-section-changer">
+				<div>
+					<section class="ui-section-active">
+						<div class="ui-content-area app-full-height">
+							No.one
+						</div>
+					</section>
+					<section>
+						<div class="ui-content-area app-full-height">
+							Two
+						</div>
+					</section>
+					<section>
+						<div class="ui-content-area app-full-height">
+							Three
+						</div>
+					</section>
+					<section>
+						<div class="ui-content-area app-full-height">
+							Four
+						</div>
+					</section>
+					<section>
+						<div class="ui-content-area app-full-height">
+							Item
```

Comment:
Five?

Reply:
This is strict from guideline. :(

Reply:
I didn't change this. When I fix it the screen will be other than in guideline.

Reply:
Fixed: Item->Five